inject/src/main/java/io/micronaut/inject/visitor/VisitorContext.java

```diff
@@ -39,10 +40,11 @@
  * @author Graeme Rocher
  * @since 1.0
  */
+@SupportedOptions({VisitorContext.MICRONAUT_PROCESSING_PROJECT_DIR})
```

Comment:
`@SupportedOptions` should be on `GraalTypeElementVisitor`

Reply:
Done.